cinema/serializer.py

```diff
@@ -0,0 +1,26 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
```

Comment:
The max_length field in your title here doesn't match the max_length in your model.